community/fulltext-index/src/main/java/org/neo4j/kernel/api/impl/fulltext/CountResult.java

```diff
@@ -0,0 +1,47 @@
+package org.neo4j.kernel.api.impl.fulltext;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+public class CountResult
+{
+    public final long count;
```

Comment:
This had to be public or Neo4j produced a `debug.log` error saying it could not access `count` and then disabled fulltext indexes.